code/game/machinery/computer/camera_circuit.dm

```diff
@@ -87,9 +87,8 @@
 		else if( href_list["auth"] )
 			var/mob/M = usr
 			var/obj/item/weapon/card/id/I = M.equipped()
-			if (istype(I, /obj/item/device/pda))
-				var/obj/item/device/pda/pda = I
-				I = pda.id
+			if (istype(I, /obj/item/modular_computer))
+				I = I.GetIdCard()
```

Comment:
`if (istype(I, /obj/item/modular_computer))`
That check is redudant and cut functionality.